tpke/src/combine.rs

```diff
@@ -2,71 +2,100 @@
 #![allow(dead_code)]
 use crate::*;
 use ark_ec::ProjectiveCurve;
+use itertools::zip_eq;
 
 pub fn prepare_combine<E: PairingEngine>(
```

Comment:
I will rename methods related to the fast variant so that they reflect their domain. Similarly how methods related to the simple variant now have `simple` in their name.

I was also considering removing the fast variant, but this is up for debate and not a subject of this PR.